package.json

```diff
@@ -14,7 +14,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "pg": ">=2.7.0-0 <5.0.0-0"
+    "pg": ">=2.7.0-0 <7"
```

Comment:
Ok. I've tested that version 6.0.0 works, so I've changed the lower bound to that